public_interface/settings.py

```diff
@@ -59,6 +59,10 @@
 
 UCLDC_DEVEL = TEMPLATE_DEBUG = bool(os.environ.get('UCLDC_DEVEL'))
 
+TEMPLATE_DIRS = (
+    os.path.join(BASE_DIR, "calisphere/templates/"),
+)
```

Comment:
This should already be in the path

Also `TEMPLATE_DIRS` is deprecated in 1.8.

I removed these lines, and it works for me.

Are we running the same version of django?

app loader on by default in 1.7
https://docs.djangoproject.com/en/1.7/ref/templates/api/#django.template.loaders.app_directories.Loader

add loader needs to be turned on in 1.8
https://docs.djangoproject.com/en/1.8/ref/templates/api/#django.template.loaders.app_directories.Loader